src/coreclr/src/debug/debug-pal/CMakeLists.txt

```
@@ -4,13 +4,29 @@ include_directories(../../pal/inc)
 
 add_definitions(-DPAL_STDCPP_COMPAT)
 
+if(FEATURE_PERFTRACING_C_LIB)
+    set(SHARED_EVENTPIPE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../../../native/eventpipe)
+    include_directories(${SHARED_EVENTPIPE_DIR})
```

top-level `configurepaths.cmake` defines this path:
```suggestion
    include_directories(${CLR_SRC_NATIVE_DIR}/eventpipe)
```

---

Good, that was not part of the commit I was working from (this was a rather new change). Thanks for pointing out! I will update to use that variable (also present in eventpipe cmake file).

---

@am11 needed to drop the conversion to path using backslash in `eng/native/configurepaths.cmake` since that will fail cmake step when trying to use that together with source files. Don't see any other of the shared paths that does this conversion to "native" path and looks like this code is the first that use CLR_SRC_NATIVE_DIR for source file location, so in order to be usable (at least for eventpipe library), I needed to remove the conversion.

---

On Windows 10, cmake's vcxproj generator was failing when I was testing with C file in my initial implementation. Later in PR, I moved code from C to header file. I think it's ok to remove that line if it is working without the conversion (and CI is green).

Aside, we could probably inline `${CLR_SRC_NATIVE_DIR}/eventpipe/` usage without introducing additional variables `SHARED_EVENTPIPE_SOURCE_PATH` and `SHARED_EVENTPIPE_DIR` etc. (unless it is done for readability).

---

I added additional variable since the value was used multiple times + similar to eventpipe cmake file (uses same variable name for same purpose) + make it more explicit/readability.